### ISSUE TITLE
Free response accessibility error

### ISSUE THREAD

**Issue by oscarlevin** [RunestoneInteractive/rs#656]
Using WAVE for checking accessibility, the free response text boxes (from a pretext reading question `<response/>`) are throwing accessibility errors.  Specifically, it complains about "Empty form label: A form label is present, but does not contain any content."

Not sure what the correct html should be, but right now the `<textarea...>` is inside a `<label>` elemement.  Do we need the label?

Is this generated by pretext or runestone?

**Comment by bnmnetp** [comment]
probably not.... but I hesitate because I know there are some things that we need to do with the multiple choice questions that involve form groups and labels.  It might just be that we need to add an aria-label to the textbox.

**Comment by oscarlevin** [comment]
There is a aria-label on the textbook already.  Here is the reference about the error that WAVE gives:

> What It Means
A form label is present, but does not contain any content.
Why It Matters
A <label> element that is associated to a form control but does not contain text will not present any information about the form control to the user.
How to Fix It
Ensure that the form label contains text that describes the function of the associated form control. Labels are not required for image, submit, reset, button, or hidden form controls. If a label is not necessary visually, a descriptive title attribute may be added to the form control.
The Algorithm... in English
A form label is present and associated with an existing form control (using for/id or surrounds the form control), but does not contain any text or images with alternative text.

**Comment by bnmnetp** [comment]
I really don't know why there was a label there.  I have removed it in the code everything works fine without it.  There is also a fieldset tag that surrounds the whole thing.  Which generates a warning about not having a legend.

**Comment by oscarlevin** [comment]
It appears that this is fixed now.  Thanks.
